projects/storefrontstyles/scss/_theme.scss

```diff
@@ -8,6 +8,15 @@
 
 //old theme to be commented out on next major or with impex
 @import 'theme/santorini/variables';
-
+@import 'theme/high-contrast/dark/variables';
+@import 'theme/high-contrast/light/variables';
+@import 'theme/high-contrast/high-contrast';
 // new theme below to uncomment on next major or with impex
 // @import 'theme/santorini-updated/variables';
+
+@mixin cx-highContrastTheme() {
+  .cx-theme-high-contrast-dark &,
+  .cx-theme-high-contrast-light & {
+    @content;
+  }
+}
```

Comment:
`minor:` I'm not sure whether this is the best place to keep this mixin. I'd consider moving it to `theme/high-contrast/mixins`.